benchmarks/quant-hela-method-comparison/scripts/04_compute_metrics.py

```
@@ -0,0 +1,581 @@
+    # 5. TMT vs LFQ correlation (PXD007683)
+    print("\n5. Computing TMT vs LFQ correlation (PXD007683)...")
+    for method in methods:
+        tmt_lfq = compute_tmt_lfq_correlation(method)
+        if tmt_lfq:
+            results["tmt_lfq"][method] = tmt_lfq
+            print(f"   {method}: R={tmt_lfq['pearson_r']:.3f}, n={tmt_lfq['n_proteins']}")
+
+    return results
+
+
+def save_results(results: dict, output_dir: Path = None):
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)